settings.py

```diff
@@ -146,6 +146,12 @@
 GITHUB_API_USER = ''
 GITHUB_API_TOKEN = ''
 
+
+AMO_LANGUAGES = AMO_LANGUAGES + ('dbr', 'dbl')
```

Comment:
can we have a `DEBUG_LANGUAGES` or similar so we're not hard-coding them in other places?